README.md

```diff
@@ -14,38 +14,36 @@ When APIs first started getting popular, a lot of companies started flopping the
 
 ## API Description
 
-Alias include: "API definition", "API contract".
+Aliases include: "API Description Document", "API Definition", "API Contract".
```

Comment:
I think this addition of API Description Document is redundant. The document is specifically the file that contains the description. Aliases for me means “other words people use which to describe the same thing.” and this is a different thing.